tests/unit/test-cmd/test.build.js

```diff
@@ -5,11 +5,17 @@ import {it, describe} from 'mocha';
 import {assert} from 'chai';
 import sinon from 'sinon';
 
-import build, {safeFileName, FileFilter} from '../../../src/cmd/build';
+import build, {safeFileName, FileFilter,
+                getDefaultLocalizedName} from '../../../src/cmd/build';
 import {withTempDir} from '../../../src/util/temp-dir';
 import {fixturePath, makeSureItFails, ZipFile} from '../helpers';
-import {basicManifest, manifestWithoutApps} from '../test-util/test.manifest';
+import {basicManifest,
```

Comment:
line breaks like this need to remain consistent. Either keep it on one line or break all elements into the next line like:
````
import {
  basicManifest,
  manifestWithoutApps,
} from '../test-util/test.manifest';
````